app/src/main/java/com/pvp/app/model/Tasks.kt

```diff
@@ -86,4 +91,19 @@ open class Task(
     override fun toString(): String {
         return "Task(description=$description, duration=$duration, id=$id, isCompleted=$isCompleted, scheduledAt=$scheduledAt, title='$title', userEmail='$userEmail')"
     }
+
+    fun scheduleReminder(context: Context, minutesToReminder: Int) {
```

Comment:
Multiple parameters could be split. If you are going to follow it, follow throughout the whole function.